src/Sulu/Component/Content/Types/ResourceLocator/Mapper/PhpcrMapper.php

```diff
@@ -290,11 +290,11 @@ public function unique($path, $webspaceKey, $languageCode, $segmentKey = null)
         return $this->isUnique($routes, $path);
     }
 
-    public function getUniquePath($path, $webspaceKey, $languageCode, $segmentKey = null)
+    public function getUniquePath($path, $webspaceKey, $languageCode, $segmentKey = null, $uuid = null)
     {
```

Comment:
Instead of adding new parameter this should use something like here and throw a deprecation message:

https://github.com/sulu/sulu/blob/ddf5a0842807aa649c97bccf8245fe6e621d81cb/src/Sulu/Bundle/MediaBundle/Media/ImageConverter/MediaImageExtractor.php#L50-L61